main.go

```diff
@@ -253,25 +274,17 @@ func runMany(dbpath string, getPort func() int, many, bucketSize, bsCon int, rel
 				totalprovs++
 			}
 		case <-reportInterval.C:
-			printStatusLine(many, start, hosts, dhts, uniqpeers, totalprovs)
+			printStatusLine(many, start, atomic.LoadInt64(&peersConnected), hyperlog.Estimate(), totalprovs)
```

Comment:
Lock when accessing the hyperlog?

Reply:
Due to how hyperlog works it woudn't do any hard (the estimate might be lower than current one if cache wasn't propageted) but I've added one.